.gitignore

```diff
@@ -133,3 +133,5 @@ typings/
 # End of https://www.gitignore.io/api/node,hugo,intellij
 
 content/tutorials/kit-assembly.files
+
+resources/
```

Comment:
??

Reply:
Whilst Hugo advocate it, I think committing the CSS build cache is a god-awful idea! I could update the `.gitignore` from gitignore.io, but adding it to the bottom felt easier

Reply:
Ah it's the css cache! I didn't know that it was in the resources folder (I don't currently use this yet).

I agree with your decision

Reply:
I also would not have known what this was. Please could we have a comment about what this is?